sessions/react-global-state/demo-start/components/Header.js

```diff
@@ -0,0 +1,28 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+const Title = styled.h1`
+  font-size: 1rem;
+  margin: 0;
+`;
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+  color: #fff;
+`;
+
+const countSum = "???";
+
+export default function Header() {
```

Comment:
It might be easier to understand when we replace a local variable within the function with a prop from the parameters

```suggestion
export default function Header() {
  const countSum = "???";
```